src/main/mainWindow.ts

```diff
@@ -235,14 +244,67 @@ function initMenuBar(win: BrowserWindow) {
             click() {
                 app.quit();
             }
+        }
+    ] satisfies MenuItemList;
+    const viewSubMenu = [
```

Comment:
whats this

Reply:
What u mean

Reply:
why are you making a new menu unrelated to the feature here

Reply:
This is the only way of detecting zoom changing as far as I know
I believe there's like window.webContents.on("zoom-change"), but it just doesn't work at all or it's for something else like trackpad zooming but im not sure 
You could detect Cmd + plus/minus but this wouldn't detect when zoom is changed from menubar but if you add a new menu it detects both that and change in menubar so it seems to be the all around best solution